jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -160,7 +160,7 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
                 false,
                 message("codemodernizer.notification.warn.invalid_project.description.reason.no_valid_files", supportedBuildFileNames.joinToString()),
                 InvalidTelemetryReason(
-                    CodeTransformPreValidationError.ProjectSelectedIsNotJava8OrJava11,
+                    CodeTransformPreValidationError.NonMavenProject,
```

Comment:
Once we started supporting other build tools like Gradle, this telemetry would likely become something more generic like `NoSupportedProjectType` if project is neither Maven or Gradle. Would it be better to start using that?

Reply:
Right now specifics around maven/gradle are useful metrics to have to determine how many projects we have or do not have with our specific requirements.